templates/csv/enmasse.clusterserviceversion.yaml

```diff
@@ -176,6 +176,34 @@ metadata:
             "name": "undefined"
           },
           "spec": {}
+        },
+        {
+          "apiVersion": "iot.enmasse.io/v1alpha1",
+          "kind": "IoTProject",
+          "metadata": {
+            "name": "iot"
+          },
+          "spec": {
+            "downstreamStrategy": {
+              "managedStrategy": {
+                "addressSpace": {
+                  "name": "iot",
+                  "plan": "standard-unlimited"
+                },
+                "addresses": {
+                  "command": {
+                    "plan": "standard-small-anycast"
+                  },
+                  "event": {
+                    "plan": "standard-small-queue"
+                  },
+                  "telemetry": {
+                    "plan": "standard-small-anycast"
```

Comment:
One thing to think about here perhaps is that these plans will not necessarily exist, unless the user is creating them by this name. With OLM there are no default plans installed so the documentation should be clear that the plans must be installed (maybe it already is).

Reply:
Done.